packages/@okta/i18n/src/properties/login.properties

```diff
@@ -711,16 +711,28 @@ oie.password = Password
 
 ## OIE external IdP
 oie.sign.in.with.idp = Sign in with {0}
+oie.footer.signInWithOtherText = Sign in using something else
```

Comment:
I am adding these strings as a part of a clean up for the footer and I have added these to a Footer section. You might be able to rebase on that PR.

Reply:
Is it merged? what's the PR #?

Reply:
Reverted my changes for this since https://github.com/okta/okta-signin-widget/pull/1236 takes care of it.